code_samples/simple_language_plugin/src/main/java/org/intellij/sdk/language/SimpleStructureAwareNavbar.java

```diff
@@ -0,0 +1,76 @@
+// Copyright 2000-2022 JetBrains s.r.o. and other contributors. Use of this source code is governed by the Apache 2.0 license that can be found in the LICENSE file.
```

Comment:
```suggestion
// // Copyright 2000-2022 JetBrains s.r.o. and contributors. Use of this source code is governed by the Apache 2.0 license.
```
Please change the header to this one in all files you modified. It can be also inserted with the "Copyright" action. 
The purpose is to fit it within the default 120 lines limit in source code files.

Reply:
Fixed.